public/app/plugins/datasource/prometheus/querybuilder/components/PromQueryEditorSelector.tsx

```diff
@@ -95,14 +86,22 @@ export const PromQueryEditorSelector = React.memo<PromQueryEditorProps>((props)
               }}
               options={promQueryModeller.getQueryPatterns().map((x) => ({ label: x.name, value: x }))}
             />
+            <QueryHeaderSwitch label="Raw query" value={query.editorPreview} onChange={onQueryPreviewChange} />
           </>
         )}
-        <QueryHeaderSwitch
-          label="Preview"
-          value={query.editorPreview}
-          onChange={onQueryPreviewChange}
-          disabled={editorMode !== QueryEditorMode.Builder}
-        />
+        {editorMode === QueryEditorMode.Builder && (
+          <BetaBadge feedbackUrl="https://github.com/grafana/grafana/pull/47653" />
```

Comment:
The url should be probably to new bug report, right? 
```suggestion
          <BetaBadge feedbackUrl="https://github.com/grafana/grafana/issues/new?assignees=&labels=type%3A+bug&template=1-bug_report.md" />
```

Reply:
@ivanahuckova 
no it should go to a github dissusion issue on the feature, that describe the feature, current limitations, our planned/hoped improvements. And asks for feedback. 

Example for what Github links to for the PR Review files pane link:

![Screenshot from 2022-04-12 15-24-30](https://user-images.githubusercontent.com/10999/163137678-24b7684f-24ec-41bd-838d-39b41a7cd998.png)

 https://github.com/github/feedback/discussions/12341